### PR TITLE
New lightweight bss theme changes, to be finalise testing by ranisa

### PR DESCRIPTION
**Pull request by ranisa-gupta16**
# feat/fix/chore: pr title

## JIRA Ticket

[[<JIRA_TICKET>](https://jira.csiro.com/browse/<JIRA_TICKET>)
](https://jira.csiro.au/browse/BSS-436)

SCREENSHOTS:

image

image

image

image

image

image

more.....

Add any additional information or context about the pull request here.

## Checklist

- [y ] I have confirmed all commits have been signed.
- [y ] I have added JSDoc style comments to any new functions or classes.
- [y ] Relevant documentation such as READMEs, guides, and class comments are updated.

### CODE REVIEW

**Comment by PeterBaker0** [comment]
On desktop I see this for the profile icon?

image

**Comment by PeterBaker0** [comment]
image
Were these small arrows always here? On the right?

**Comment by PeterBaker0** [comment]
I think the bubble theme is broken - namely in sync_switch it expects a `theme.palette.icon.main` which is undefined for bubble. 
sync_switch.tsx
```
<Box>
          <FormControlLabel
            sx={{mr: 0, md: 2}}
            control={
              <Switch
                checked={project.sync}
                disabled={false}
                onClick={handleOpen}
                sx={{
                  '& .MuiSwitch-switchBase.Mui-checked': {
                    **color: theme.palette.icon.main,**
                  },
```

**Review by PeterBaker0 — CHANGES_REQUESTED** [review]
My profile icon shows as a white background, and bubble is throwing an error at the moment. Otherwise looks good!

File: app/.env.dist [hunk 1]
```diff
@@ -15,8 +15,8 @@ VITE_DEBUG_APP=true
 VITE_PRODUCTION_BUILD=false
 # Set to true for verbose debug messages from PouchDB
 VITE_POUCHDB_DEBUG=false
-# The theme to use for the app, 'default' or 'bubble'
-VITE_THEME='default'
+# The theme to use for the app, 'default' or 'bubble' or 'bssTheme'
+VITE_THEME='bssTheme'
```
[the remark this answers is not in the document]
Comment (ranisa-gupta16) [reply app/.env.dist:19]: Yes i'll set this to default, if we need bss we'll switch it.

File: app/public/manifest.json
[the remark this answers is not in the document]
Comment (ranisa-gupta16) [reply app/public/manifest.json:1]: Ah no, I need to uncommit this.

File: app/src/gui/components/notebook/record_table.tsx
[the remark this answers is not in the document]
Comment (ranisa-gupta16) [reply app/src/gui/components/notebook/record_table.tsx:1]: yes I agree. It's the best case approach right now to have those theming in. We'll need to do some improvisations moving ahead.

File: app/src/gui/components/notebook/settings/index.tsx [hunk 2]
```diff
@@ -111,6 +112,15 @@ export default function NotebookSettings(props: {uiSpec: ProjectUIModel}) {
                   control={
                     <Switch
                       checked={isSyncing}
+                      sx={{
+                        '& .MuiSwitch-switchBase.Mui-checked': {
+                          color: theme.palette.icon.main,
+                        },
+                        '& .MuiSwitch-switchBase.Mui-checked + .MuiSwitch-track':
```
[the remark this answers is not in the document]
Comment (ranisa-gupta16) [reply app/src/gui/components/notebook/settings/index.tsx:119]: I find it from chrome dev tools, 

whatever section I'm looking at , I get to its root 
image
image

and if I need more info I refer to Mui CSS Styles API - here's the link https://mui.com/system/styles/api/

File: app/src/gui/themes/bssTheme/appBar.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import {createTheme} from '@mui/material';
+import {createUseStyles as makeStyles} from 'react-jss';
+
+const theme = createTheme();
+
+const drawerWidth = 240;
+
+// bss theme related elements to be tested  - @todo RAnisa
```
[the remark this answers is not in the document]
Comment (ranisa-gupta16) [reply app/src/gui/themes/bssTheme/appBar.tsx:8]: This is done, updating this todo.
Updating this todo.

**Comment by ranisa-gupta16** [comment]
> image Were these small arrows always here? On the right?

Yes this is preexisting , might try to fix this

**Comment by ranisa-gupta16** [comment]
> On desktop I see this for the profile icon?
> 
> image

Ammm, I don;'t see this, may be hard refresh and see, I see sign in in both mobile and desktop versions for both the themes.
image

image

**Comment by ranisa-gupta16** [comment]
> I think the bubble theme is broken - namely in sync_switch it expects a `theme.palette.icon.main` which is undefined for bubble. sync_switch.tsx
> 
> ```
> <Box>
>           <FormControlLabel
>             sx={{mr: 0, md: 2}}
>             control={
>               <Switch
>                 checked={project.sync}
>                 disabled={false}
>                 onClick={handleOpen}
>                 sx={{
>                   '& .MuiSwitch-switchBase.Mui-checked': {
>                     **color: theme.palette.icon.main,**
>                   },
> ```
Bubble theme have had some issues. That was the reason we've created a new theme and not using bubble anymore.
We might have to fix bubble in future may be or that'll be removed down the line.

File: app/.env.dist (hunk 1 above)
[the remark this answers is not in the document]
Comment (ranisa-gupta16) [reply app/.env.dist:19]: Bubble theme is not working that was the reason of creating a new theme.
I believe we would need to either spend time fixing that theme in future or remove it.
I'll add a ticket to look into that in backlog.

File: app/src/gui/pages/record-create.tsx [hunk 4]
```diff
@@ -280,10 +280,16 @@ function DraftEdit(props: DraftEditProps) {
...
+                style: {
+                  backgroundColor: theme.palette.secondary.contrastText,
+                },
+              }}
+              sx={{backgroundColor: theme.palette.background.tabsBackground}}
               textColor="secondary"
             >
               <Tab label="Create" value="1" sx={{color: '#c2c2c2'}} />
-              <Tab label="Meta" value="2" sx={{color: '#c2c2c2'}} />
+              <Tab label="Meta" value="2" sx={{color: '#C13E3EFF'}} />
```
[the remark this answers is not in the document]
Comment (ranisa-gupta16) [reply app/src/gui/pages/record-create.tsx:292]: Yup this was fixed based on theming, needed to remove these.

**Comment by ranisa-gupta16** [comment]
> > image Were these small arrows always here? On the right?
> 
> Yes this is preexisting , might try to fix this

Ahh I see, there was a unnecessary scroll added for this which is why it was displaying those arrows.
Fixed it.

**Comment by ranisa-gupta16** [comment]
@PeterBaker0 , I have fixed the issues, resolved all conversations for this PR and have also added comments on the ones that doesn't need fixing.
Please review this when you get a chance.  :-)

**Comment by PeterBaker0** [comment]
> > I think the bubble theme is broken - namely in sync_switch it expects a `theme.palette.icon.main` which is undefined for bubble. sync_switch.tsx
> > ```
> > <Box>
> >           <FormControlLabel
> >             sx={{mr: 0, md: 2}}
> >             control={
> >               <Switch
> >                 checked={project.sync}
> >                 disabled={false}
> >                 onClick={handleOpen}
> >                 sx={{
> >                   '& .MuiSwitch-switchBase.Mui-checked': {
> >                     **color: theme.palette.icon.main,**
> >                   },
> > ```
> 
> Bubble theme have had some issues. That was the reason we've created a new theme and not using bubble anymore. We might have to fix bubble in future may be or that'll be removed down the line.

as discussed, theme should either work, or be removed - it's okay if the colours are not good.

**Comment by PeterBaker0** [comment]
> @PeterBaker0 , I have fixed the issues, resolved all conversations for this PR and have also added comments on the ones that doesn't need fixing. Please review this when you get a chance. :-)

See fix I pushed for sign in styling - all of your screenshots show logged out

**Review by PeterBaker0 — APPROVED** [review]
Once bubble theme functions again - even if it looks bad, is fine with me. I have pushed a proposed fix for the sign in styling.

**Pull request merged**